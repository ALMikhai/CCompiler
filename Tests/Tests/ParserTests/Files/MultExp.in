a * b++ / c % n.m
