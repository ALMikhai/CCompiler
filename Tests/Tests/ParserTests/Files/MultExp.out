\-MOD
  |-DIV
  | |-MULT
  | | |-a
  | | \-POSTFIX INC
  | |   \-b
  | \-c
  \-.
    |-n
    \-m

